actions/user_actions.jsx

```diff
@@ -419,3 +421,11 @@ export function autoResetStatus() {
         return userStatus;
     };
 }
+
+export function trackDMGMOpenChannels() {
+    return async (doDispatch, doGetState) => {
+        const state = doGetState();
+        const channels = getDirectChannels(state);
```

Comment:
The selector does filter the auto closed, manual closed DM's and GM's but the comment seems to be wrong or outdated https://github.com/mattermost/mattermost-redux/blob/master/src/selectors/entities/channels.ts#L1043

Reply:
When we're moving these into the webapp, we really need to take a look at what selectors we actually need and which aren't used any more. We've got probably 3 or 4 versions of these selectors that return channels where some are filtered in different ways and some have extra data added to the channel. That code is kind of a mess, so I'm not surprised the comments could be out of date